modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -165,6 +165,117 @@ inline void copyFromIE(const IE::Blob::Ptr &blob, MatType &mat) {
     }
 }
 
+#if INF_ENGINE_RELEASE < 2020000000  // < 2020.1
+// Load extensions (taken from DNN module)
+std::vector<std::string> getExtensions(const cv::gapi::ie::detail::ParamDesc& params) {
```

Comment:
I'd additionally wrap all these functions into a namespace like `gapi::ie::wrap` to let the caller code clearly see where it all comes from

Reply:
> @dkurt does DNN module plan to support that?

We just integrated in once and it's OK to keep but not sure that it's suitable to add as new functionality.

Reply:
Wrapped in a namespace